mm2src/coins_activation/src/prelude.rs

```diff
@@ -72,27 +73,42 @@ pub enum CoinConfWithProtocolError {
     UnexpectedProtocol { ticker: String, protocol: CoinProtocol },
 }
 
+/// Determines the coin configuration and protocol information for a given coin or NFT ticker.
+/// In the case of NFT ticker, it's platform coin config will be returned.
 #[allow(clippy::result_large_err)]
 pub fn coin_conf_with_protocol<T: TryFromCoinProtocol>(
     ctx: &MmArc,
     coin: &str,
 ) -> Result<(Json, T), MmError<CoinConfWithProtocolError>> {
-    let conf = coin_conf(ctx, coin);
+    let (conf, coin_protocol) = match Chain::from_nft_ticker(coin) {
```

Comment:
Is there a reason for keeping supported nfts inside the code and not in the config?

Reply:
> nfts inside the code and not in the config

Keeping nfts in the coin conf is what we are trying to avoid as it is actually not a coin. 
We should build nft Coin types from platform coin. If parent config was changed, there is no need to additionally change nft "coin" in config file. Moreover, as we have common code base for EVM based coins like eth, bnb, avax etc there is no need to create config for each of their nfts.

Reply:
I think I was a bit wrong in suggesting not adding NFTs in coin configs, if moralis supported more EVM chains or we moved away from moralis for a web3 native implementation (if possible in the future), it would be easier to add new chains and their NFTs in config without requiring a PR in komodefi. I guess one way to do this in the future is to have a field in platform coin configuration for if they support NFT or not, and we enable global NFT using the platform ticker and append `NFT_` prefix to it. This is not needed at all for now, but if it was requested in the future we can do it.

Reply:
> it would be easier to add new chains and their NFTs in config without requiring a PR in komodefi.

hmm not sure that if we start to use `nft_support:true` in coin config, we wont need to open PR at all. 
We still will have to add `nft_infos` field in coin type structure, for example `TendermintCoinImpl`.Or we probably need to develop new (a really) global nft type
```rust
pub struct GlobalNft(Arc<GlobalNftImpl>);
```

and make it general for all coins. hmm or update implementation of `struct NftCtx`, so it will have similar behavior as `struct CoinsContext` but for nfts.

as for now, if moralis add new evm chain support, we just need to add this match in `Chain`. well yeah, I was wrong that we wont need to  do anything. well still it is easy update.

But if we refactor nft code and make it able to support any coin type (struct GlobalNft or struct NftCtx, or maybe even both), this way yeah, it will be enough to add `nft_support:true` or `nft_support:false` in coin config.
We will see :) of course I will support the idea of such refactoring if it will be necessary.

Reply:
> as for now, if moralis add new evm chain support, we just need to add this match in Chain. well yeah, I was wrong that we won't need to do anything. well still it is easy update.

Yeah, I meant for EVM chains only (e.g. [Moonbeam](https://github.com/KomodoPlatform/coins/blob/3178b63d4dab04a881c2bba4b6a886a79d794936/coins#L8313-L8329)), if moralis added support for it, we would only need `nft_support:true` in coin config.